src/backend/joanie/core/views.py

```diff
@@ -37,3 +38,21 @@ class DebugMailSuccessPaymentViewTxt(DebugMailSuccessPayment):
     in text format"""
 
     template_name = "mail/text/order_validated.txt"
+
+
+class BackOfficeRedirectView(RedirectView):
```

Comment:
:ok_hand: 
Another way would be to do it in nginx. I think that's how we do it between richie and edxapp.

Reply:
Yup indeed. The only drawback is that this feature will not be available with the development service.